contrib/video_encoding_tester.py

```diff
@@ -155,7 +155,7 @@ def download_original(output_dir, youtube_id, video_format):
     fpath = expected_path.parent.joinpath(expected_path.stem)
     audext, vidext = {"webm": ("webm", "webm"), "mp4": ("m4a", "mp4")}[video_format]
     subprocess.run(
-        [
+        [  # noqa: S607 # nosec B607
```

Comment:
I wasn't aware that executing with `/usr/bin/env` avoids those warnings.
Aside executing with it, is there any other change needed to make it work? (otherwise I see no reason not to do it right now)

Is it better to execute with `/usr/bin/env` or simply pass the whole path `/usr/local/bin/youtube-dl`?

Reply:
The full path is not flexible. If you have `/usr/local/bin/youtube-dl` hard coded, it will work inside the container but locally, chances are youtube-dl binary is elsewhere. `["/usr/bin/env", "youtube-dl"]` uses the `PATH` variable

Reply:
Yes, but then using `/usr/bin/env` is just a trick which is not detected by quality tools which make them believe we use a fully qualified path instead of a partial one, so there is not much difference with the `# noqa: S607 # nosec B607` hint from my PoV. Do I miss something?

Reply:
> Do I miss something?
My first comment, where mentioned I'm fine with both as long as we stick to it. I don't want inconsistency accros repos (or even lines/files).
Please confirm we're only using one method in the repo

Reply:
Let's use `/usr/bin/env`, it seems to be the popular option across repos.